bottlecap/Cargo.toml

```diff
@@ -16,13 +18,13 @@ protobuf = { version = "3.4.0", default-features = false }
 serde = { version = "1.0.197", features = ["derive"], default-features = false }
 serde_json = { version = "1.0.116", features = ["alloc"], default-features = false }
 thiserror = { version = "1.0.58", default-features = false }
-tracing = {version = "0.1.40", default-features = false }
+tracing = { version = "0.1.40", default-features = false }
 tracing-core = { version = "0.1.32", default-features = false }
 tracing-log = { version = "0.2.0", default-features = false, features = ["std", "log-tracer"] }
 tracing-subscriber = { version = "0.3.18", default-features = false, features = ["fmt", "env-filter"] }
 ureq = { version = "2.9.7", features = ["tls", "json"], default-features = false }
 ustr = { version = "1.0.0", default-features = false }
-
+tokio = { version = "1.37.0" }
```

Comment:
Are we sure we want everything included here?

Reply:
Probably not, this PR is still very much WIP now, since 2 approaches are being checked.
I will convert it to draft and clean it up before we re evaluate merging it